### PR TITLE
Added path of the extension directory, removed some typo and Resolved the issue(#803).

### PR DESCRIPTION
**Pull request by PratikDevlekar**
Added path of the extension directory and removed some typo.

### CODE REVIEW

File: docs/extensions/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ To get started with development, one should usually follow the steps in "[Create
 conventions closely resemble those of CiviCRM Core and of CiviCRM-Drupal modules. Module extensions are fully supported in CiviCRM 4.2+.
 
 ### 3. Publish
-The CiviCRM ecosystem is built on the belief that non-profit organizations can serve themselves best by collaborating in development of their data-management applications. As staff, volunteers, and consultants for non-profit organizations, we can share our new enhancements and extensions -- and build a richer whole for the entire ecosystem.
+The CiviCRM ecosystem is built on the belief that non-profit organizations can serve themselves best by collaborating in development of their data-management applications. As a staff, volunteers, and consultants for non-profit organizations, we can share our new enhancements and extensions -- and build a richer whole for the entire ecosystem.
```
Comment (mattwire) [inline docs/extensions/index.md:53]: @PratikDevlekar This was correct before (staff is multiple people too).

**Comment by mattwire** [comment]
Hey @PratikDevlekar This looks good, can you just amend the last bit per my comment above.

File: docs/extensions/index.md (hunk 1 above)
Comment (PratikDevlekar) [reply docs/extensions/index.md:53]: Yes sure.

**Comment by PratikDevlekar** [comment]
I had made that minor change and also resovled another issue too(https://github.com/civicrm/civicrm-dev-docs/issues/803).

**Comment by mattwire** [comment]
Thanks @PratikDevlekar

**Pull request merged**